CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+# Contributing
+You can contribute to this project in many different ways.  **We appreciate all kinds of help, so thank you!** 
+
+## Issues and Enhancements
+Please let us know via our [issue tracker](https://github.com/AdoptOpenJDK/openjdk-build/issues) if you find a problem, even if you don't have a fix for it.  The ideal issue report should be descriptive, and where possible include the steps we can take to reproduce the problem for ourselves.
+
+If you have a proposed fix for an issue, or an enhancement you would like to make to the code please describe it in an issue, then send us the code as a [Github pull request](https://help.github.com/articles/about-pull-requests) as described below.
+
+## Pull requests
+We use GitHub's pull requests (PRs) as the primary way to accept contributions to the project.  That means we assume you have followed the usual procedure and forked the project repository, cloned your fork, created a new branch for your contribution, and pushed one or more commits to your branch.  There are many [on-line guides](https://guides.github.com/activities/forking/) that will help you with these steps.
```

Comment:
Minor: add labels to the PR to help identify the type of change (helps further if references to the issue or issues being resolved is mentioned in the description of the PR and/or in the commit messages).